packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -14,7 +15,10 @@ export class GraphQLRequestClient {
    * @param {string} query graphql query
```

Comment:
Update the param to reflect new allowed `DocumentNode`. Funny, I made the [same change in my branch](https://github.com/Sitecore/jss/commit/cc56d97c16cfad3a0e6f04e9d0e5732bdb8f7202#diff-ce9b582943d66dd42dbe58b3b6fe05f0c970ea184d8d41eddd5b93529b2906eaR23) 😄 
```suggestion
   * @param {string | DocumentNode} query graphql query
```

Reply:
Eagle eye 👍  I added